.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "src/tensorizer"]
-	path = src/tensorizer
-	url = https://github.com/coreweave/tensorizer.git
+[submodule "src/proto/tensorizer"]
+	path = src/proto/tensorizer
```

Comment:
If we do this, we will need to adjust the library paths in things that use this.

Reply:
Do you know of something impacted? Golang based fetch from tensorizer directly, and does not appear to be an issue with the TS/JS partially due to it not working before.  Python would be about the only one to be concerned with this change from what I can tell depending on how the project was setup.